src/logger/loggerTaskEx.c

```diff
@@ -243,8 +247,7 @@ void loggerTaskEx(void *params)
                 }
 
                 /* send the sample on to the telemetry task(s) */
-                if (sampledRate >= telemetrySampleRate ||
-                    currentTicks % telemetrySampleRate == 0)
+                if (!isHigherSampleRate(sampledRate, telemetrySampleRate))
```

Comment:
Thanks. Please attach it to a r/2.8.6 branch and we can move forward with the 2.8.5 release. 

Need to do more analysis on the impact of this, but as I remember it's to prevent cases where we may not queue any telemetry messages if a user selects a certain combinations of sample rates.